### PR TITLE
[MM-48898] Implement info prompt for approaching max call duration

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

PR implements an info prompt that's showed to the host when the current call recording duration is approaching (10 minutes threshold) the configured limit.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-48898

### CODE REVIEW

File: server/configuration.go [hunk 1]
```diff
@@ -73,6 +73,12 @@ type clientConfig struct {
 	MaxRecordingDuration *int
 }
 
+const (
+	defaultRecDurationMinutes = 60
+	minRecDurationMinutes     = 15
+	maxRecDurationMinutes     = 180
```
Comment (streamer45) [inline server/configuration.go:79]: @justinegeffen I am temporarily setting these as min/max limits. We can discuss tomorrow in our call if they need tuning and then can be added to the documentation.

File: webapp/src/components/expanded_view/recording_info_prompt.tsx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+                        icon='alert-outline'
+                        style={{
+                            fontSize: 18,
+                        }}
+                    />
+                }
+                iconFill='rgb(var(--dnd-indicator-rgb))'
+                iconColor='rgb(var(--dnd-indicator-rgb))'
+                header={`Your recording will end in ${getMinutesLeftBeforeEnd()} minutes`}
+                body={`Calls can be recorded for up to ${props.recordingMaxDuration} minutes.`}
```
Comment (streamer45) [inline webapp/src/components/expanded_view/recording_info_prompt.tsx:77]: @justinegeffen Please help with the right content to show here.

**Review by cpoile — APPROVED** [review]

**Comment by streamer45** [comment]
Given Justine is out I am going to merge this and we can review it post release if necessary.

File: server/configuration.go (hunk 1 above)
Comment (justinegeffen) [reply server/configuration.go:79]: Brilliant, that works for me. Thanks.
(i missed the meeting as I was in surgery, sorry!)

File: webapp/src/components/expanded_view/recording_info_prompt.tsx (hunk 2 above)
Comment (justinegeffen) [reply webapp/src/components/expanded_view/recording_info_prompt.tsx:77]: This reads well to me. I wonder if we should switch the body and header?

header={`Calls can be recorded for up to ${props.recordingMaxDuration} minutes.`}
body={`Your recording will end in ${getMinutesLeftBeforeEnd()} minutes`}

File: server/configuration.go (hunk 1 above)
Comment (streamer45) [reply server/configuration.go:79]: No worries, hope everything went well :)

File: webapp/src/components/expanded_view/recording_info_prompt.tsx (hunk 2 above)
Comment (streamer45) [reply webapp/src/components/expanded_view/recording_info_prompt.tsx:77]: Yep, we can do that. Let me update on the main PR. Thanks!

**Pull request merged**